engine/crates/federated-graph/src/federated_graph.rs

```diff
@@ -185,6 +188,15 @@ pub struct FieldRequires {
     pub fields: FieldSet,
 }
 
+/// Represents an `@override(graph: .., from: ...)` directive on a field in a subgraph.
+#[derive(serde::Serialize, serde::Deserialize, Clone)]
+pub struct Override {
+    pub graph: SubgraphId,
+    /// Points to a subgraph referenced by name, but this is _not_ validated to allow easier field
+    /// migrations between subgraphs.
+    pub from: StringId,
```

Comment:
Not sure to follow the logic here. If the `from` graph doesn't exist, the overrides shouldn't be present in the federated graph imho. There's nothing else to do with it at runtime. 🤷

Reply:
I agree from the perspective of the router, but we need to still have them there in order to roundtrip to federation SDL, I checked and other implementations allow schemas that override fields that do not exist in the `from` schema. Alternatively, this could be an enum of `enum OverrideSource { Existing(SubgraphId), Missing(StringId) }`.

Reply:
or a separate vec of unresolved overrides

Reply:
Good point, missed that. Well if it's easy for you would be better to have that enum IMHO, but not a big deal to do the mapping later.

Reply:
I'll do the enum, should be very easy.

Reply:
Done in 86ae7405b25918442d1468f6d1cd48990a613d15

Reply:
Thanks!